initializers/services.yml

```diff
@@ -0,0 +1,10 @@
+# - name: SSH
+#   protocol: TCP
+#   ports:
+#     - 22
+#   virtual_machine: virtual machine 1
+# - name: FTP
+#   protocol: TCP
```

Comment:
Is UDP an option here? It's be great to know the options either by listing them as a comment or by showing them in various examples.

e.g. DNS is a UDP protocol (port 53, although it's also a TCP protocol.)

Reply:
Great idea.  I updated